Source/Serilog.Enrichers.Span/ActivityEnricher.cs

```diff
@@ -33,15 +33,25 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
                 AddSpanId(logEvent, activity);
                 AddTraceId(logEvent, activity);
                 AddParentId(logEvent, activity);
+                AddTags(logEvent, activity);
```

Comment:
This is a good idea but we should make it configurable.

Reply:
thank you for your opinion.

I will be back.